[2024-04-18T00:32:24.231+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-18T00:32:24.262+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dagPartition.transform manual__2024-04-18T00:26:20.462979+00:00 [queued]>
[2024-04-18T00:32:24.272+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dagPartition.transform manual__2024-04-18T00:26:20.462979+00:00 [queued]>
[2024-04-18T00:32:24.273+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-04-18T00:32:24.294+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): transform> on 2024-04-18 00:26:20.462979+00:00
[2024-04-18T00:32:24.302+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1944) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-18T00:32:24.305+0000] {standard_task_runner.py:63} INFO - Started process 1946 to run task
[2024-04-18T00:32:24.304+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dagPartition', 'transform', 'manual__2024-04-18T00:26:20.462979+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/PartitionLocation.py', '--cfg-path', '/tmp/tmp69mtdkpt']
[2024-04-18T00:32:24.306+0000] {standard_task_runner.py:91} INFO - Job 140: Subtask transform
[2024-04-18T00:32:24.382+0000] {task_command.py:426} INFO - Running <TaskInstance: dagPartition.transform manual__2024-04-18T00:26:20.462979+00:00 [running]> on host 0761b779d565
[2024-04-18T00:32:24.554+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dagPartition' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2024-04-18T00:26:20.462979+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-18T00:26:20.462979+00:00'
[2024-04-18T00:32:24.557+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-18T00:32:24.612+0000] {logging_mixin.py:188} INFO -                                      id  ... dt_process
0  5128df48-79fc-4f0f-8b52-d06be54d0cec  ...   20240418
1  9c5a66c8-cc13-416f-a5d9-0a769c87d318  ...   20240418
2  34e8c68b-6146-453f-a4b9-1f6cd99a5ada  ...   20240418
3  ef970757-fe42-416f-931d-722451f1f59c  ...   20240418
4  6d14b220-8926-4521-8d19-b98a2d6ec3db  ...   20240418

[5 rows x 17 columns]
[2024-04-18T00:32:24.613+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-18T00:32:24.614+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-18T00:32:24.637+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dagPartition, task_id=transform, execution_date=20240418T002620, start_date=20240418T003224, end_date=20240418T003224
[2024-04-18T00:32:24.681+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-18T00:32:24.719+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-18T00:32:24.725+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
